tests/lava/lib/optimization/solvers/generic/test_solver_cpu_backend.py

```diff
@@ -131,19 +131,112 @@ def test_state_tracking(self):
         )
 
 
-def solve_workload(problem, reference_solution, noise_precision=5,
-                   noise_amplitude=1, on_tau=-3):
+class TestOptimizationSolverQP(unittest.TestCase):
+    def setUp(self) -> None:
+        root = os.path.dirname(os.path.abspath(__file__))
+        qp_data = np.load(root + "/data/qp/ex_qp_small.npz")

```

Comment:
I would suggest not using upper case for variable names

Reply:
I also don't like that practice but in this case for QP it makes life easier and popular solvers also do it.